pkg/cmd/server/start.go

```diff
@@ -358,6 +358,8 @@ func start(cfg *config, args []string) error {
 			KubernetesAddr:       cfg.KubernetesAddr.URL.String(),
 			KubernetesPublicAddr: k8sPublicAddr.URL.String(),
 
+			LogoutURI: env("OPENSHIFT_LOGOUT_URI", ""),
```

Comment:
yeah, all the auth config is env only until we think through how we really want it structured/specified